src/apps/Permissions/Home/AllPermissions.js

```diff
@@ -0,0 +1,189 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  DropDown,
+  GU,
+  IconSearch,
+  TextInput,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import EmptyBlock from '../EmptyBlock'
+import EmptyFilteredPermission from '../EmptyFilteredPermissions'
+import PermissionsView from '../PermissionsView'
+
+const ENTITY_TYPES = ['All entities', 'Accounts', 'Apps']
+
+function AllPermissions({ loading, permissions, onManageRole }) {
+  const [selectedEntityType, setSelectedEntityType] = useState(-1)
+  const [searchTerms, setSearchTerms] = useState('')
```

Comment:
⚠️ not sure what happened but when I submitted a couple of odd search strings this broke and rendered the error screen: '  a  b' and 'bbb'

Reply:
@AquiGorka Any idea on how to reproduce this?